components/Landing/HeroMoment/LandingHeroMomentComponent.vue

```diff
@@ -2,37 +2,65 @@
   <article class="hero-moment">
     <LandingHeroMomentGrid>
       <div class="cds--grid hero-moment__container">
-        <LandingHeroMomentVersionInfo
-          class="hero-moment__version-info"
-          :version="version"
-        />
         <div class="cds--row">
           <h1
-            class="cds--col-md-3 cds--col-lg-5 cds--col-xlg-6 cds--col-max-7 hero-moment__title"
+            class="cds--col-md-4 cds--col-lg-5 cds--col-xlg-5 cds--col-max-7 hero-moment__title"
           >
-            Your open-source toolkit for useful quantum computing
+            <img
+              src="/images/landing-page/logo_wordmark.svg"
+              class="hero-moment__title__logo"
+            />
           </h1>
         </div>
         <div class="cds--row">
-          <p
-            class="cds--col-sm cds--col-md-3 cds--col-lg-5 cds--col-xlg-4 hero-moment__description"
+          <div
+            class="cds--col-sm cds--col-md-4 cds--col-lg-5 cds--col-xlg-5 hero-moment__description"
           >
-            Qiskit {{ qiskitPronunciation }} is an open-source SDK for working
-            with quantum computers at the level of pulses, circuits, and
-            application modules.
-          </p>
+            <h3 class="hero-moment__description__title">
+              <strong>{{ qiskitPronunciation }}</strong> <em>noun, software</em>
+            </h3>
+            <hr class="hero-moment__description__divider" />
+            <h3>
+              <ol class="hero-moment__description__content">
+                <li>open-source toolkit for useful quantum computing.</li>
+                <li>production-ready circuit compiler.</li>
+              </ol>
+            </h3>
+          </div>
         </div>
         <UiCta
           class="hero-moment__cta"
           :label="getStartedLink.label"
           :segment="getStartedLink.segment"
           :url="getStartedLink.url"
         />
+        <LandingHeroMomentVersionInfo
+          class="hero-moment__version-info"
+          :version="version"
+        />
         <img
           class="hero-moment__container__image"
-          src="/images/landing-page/new-hero-illustration.png"
+          src="/images/landing-page/cryo.png"
           alt="A visual composition of a sketched a quantum computer backdrop, a progress bar indicating a running job, and a laptop screen displaying the text 'running job...'"
         />
+        <div class="hero-moment__container__laptop">
+          <img
+            class="hero-moment__container__laptop__frame"
+            src="/images/landing-page/laptop.png"
+            alt="Laptop"
```

Comment:
Could you add more description to the `alt`? maybe we can reference to the code too